File: ProjectHorizon.ApplicationCore/Constants/AuditLogCategory.cs
namespace ProjectHorizon.ApplicationCore.Constants
{
    public class AuditLogCategory : BaseEnum<AuditLogCategory>
    {
        public const string AllCategories = "All Categories";
        public const string PublicRepository = "Public Repository";
        public const string PrivateRepository = "Private Repository";
        public const string Approvals = "Approvals";
        public const string UserManagement = "User Management";
        public const string NotificationSettings = "Notification Settings";
        public const string Integrations = "Integrations";
        public const string SingleSignOn = "Single Sign-On";
        public const string Subscription = "Subscription";
        public const string AssignmentProfiles = "Assignment Profiles";
        public const string ShopRequests = "Shop Requests";
        public const string DeploymentSchedules = "Deployment Schedules";
    }
}   

File: ProjectHorizon.WebAPI/ClientApp/src/app/constants/audit-log-category.ts
export const AuditLogCategories = [
  'All Categories',
  'Public Repository',
  'Private Repository',
  'Approvals',
  'User Management',
  'Notification Settings',
  'Integrations',
  'Single Sign-On',
  'Subscription',
  'Assignment Profiles',
  'Shop Requests',
  'Deployment Schedules',
] as const;
export type AuditLogCategory = typeof AuditLogCategories[number];
